website/software_install.md

```diff
@@ -9,8 +9,17 @@ hascode = true
 
 ## Opening and running the Jupyter Julia notebook
```

Comment:
@mauro3 Would be good to add infos here on how students can deal with notebooks, in particular for the homework. One should describe workflow on how to open the notebook, save a personal copy, edit it, then save the folder containing the notebooks to then upload it to Moodle.